src/In.ProjectEKA.FHIRHip/DataFlow/Model/NetworkData.cs

```diff
@@ -0,0 +1,17 @@
+namespace In.ProjectEKA.DefaultHip.DataFlow.Model
+{
+    using HipLibrary.Patient.Model;
+    public class NetworkData
+    {
+        public string CareContext { get; set; }
+        public HiType HiType { get; set; } 
+        public string Data { get; set; }
```

Comment:
should we consider naming it as 'DataInFHIRFormat'/'FHIRData'?

Reply:
I am using the same name mentioned in Chat you shared

Reply:
Can we change it to be more specific?

Reply:
done